docs/release-notes/cloud/cloud-2023-08.md

```diff
@@ -0,0 +1,80 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud August 2023.
+included_jira_versions: ['2023.08']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.6.39
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.8.2
+---
+
+# Cloud August 2023
+
+These release notes are for the Codacy Cloud updates during August 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added [support for Trivy](../../getting-started/supported-languages-and-tools.md) to detect committed secrets, IaC misconfigurations, and other security risks. (TAROT-2290)
+-   The CSV file exported by Security and risk management now includes three new columns: file path, line number, and detected pattern. (TAROT-2203)
+-   When developers commit from Bitbucket, now Codacy automatically associates all commit email addresses belonging to the same Bitbucket user with a single Codacy committer. This reduces the number of duplicate seats for Bitbucket organizations when developers don’t log in to Codacy. (PLUTO-652)
+-   The [getRepositoryWithAnalysis API endpoint](https://api.codacy.com/api/api-docs#getrepositorywithanalysis) now includes data about coverable and covered lines for a given repository. (ALA-254)
+-   Pylint now supports Python 3.11 syntax. (TS-467)
+
+## Bug fixes
+
+-   Fixed an issue occurring for some Enterprise plans that caused the People page to show fewer emails than expected for committers who logged in to the Codacy app. (PLUTO-691)
```

Comment:
@nicklem I replaced "members" with an expression our users can better understand. Let me know if it needs some grammar tweaking. 😉